File: awari/exerciciosDeFundamentoDeJava-e-Py/TESTE.js
function soma (a,b) {
    console.log(a + b)
}
soma(10, 52)

// Definir a lista de números
var lista = [1, 2, 3, 45, 68, 58];

// Definir um valor inicial para o maior número
var maior = lista[0];

// Usar um loop for para percorrer cada elemento da lista
for (var i = 0; i < lista.length; i++) {
  // Verificar se o elemento atual é maior do que o maior número
  if (lista[i] > maior) {
    // Atualizar o valor do maior número
    maior = lista[i];
  }
}

// Mostrar o resultado
console.log("O maior número é " + maior);

File: awari/exerciciosDeFundamentoDeJava-e-Py/teste.py
def soma(a,b):
    print(a + b)
soma(22, 11)

for i in range(2, 9, 2): 
    for j in range (i, 5): 
        for k in range(1, j+1, i): 
            print(i, ", ", j, ", ", k)
